lib/src/gantt_default_day_header.dart

```diff
@@ -29,7 +29,7 @@ class GanttChartDefaultDayHeader extends StatelessWidget {
       ),
       child: Center(
         child: Text(
-          weekDay.symbol,
+          '${weekDay.symbol}\n${date.day}',
```

Comment:
wouldn't it make sense to just append it instead of a new line ?
e.g.: 
`SU 13` vs 
```
SU
13
```
this also saves the 10 pixels of added height
AND users can customize this anyway to suit their needs

Reply:
![image](https://user-images.githubusercontent.com/98157423/211558407-43231ea2-73f4-4132-b630-c05fa412a053.png)

If I append then sometimes space isn't sufficient and UI automatically add a new line adding incoherence
It sure might be possible to dig this further to that fonts are resized accordingly, but I do not master this,

Also, it looks clean to read days number on a horizontal line,
But this is up to you

Reply:
well, you have a valid point, I think it's better if the number is on a new line